src/applications/appeals/996/tests/components/ConfirmationPagev2.unit.spec.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { expect } from 'chai';
+
+import {
+  $,
+  $$,
+} from '@department-of-veterans-affairs/platform-forms-system/ui';
+
+import ConfirmationPageV2 from '../../components/ConfirmationPageV2';
+import maxData from '../fixtures/data/maximal-test-v2.5.json';
+
+const getData = (customData = {}) => ({
+  user: {
+    profile: {
+      userFullName: {
+        first: 'Michael',
+        middle: 'Thomas',
+        last: 'Wazowski',
+        suffix: 'Esq.',
+      },
+      dob: '1990-02-03',
+    },
+  },
+  form: {
+    formId: '20-0996',
+    submission: {
+      response: new Date().toISOString(),
+    },
+    data: {
+      ...maxData.data,
+      ...customData,
+    },
+  },
+});
+
+describe('ConfirmationPageV2', () => {
+  const middleware = [thunk];
+  const mockStore = configureStore(middleware);
+
+  it('should render with no data', () => {
+    const { container } = render(
+      <Provider store={mockStore({})}>
+        <ConfirmationPageV2 />
+      </Provider>,
+    );
+    expect($('va-alert[status="success"]', container)).to.exist;
+
+    const items = $$('.dd-privacy-hidden[data-dd-action-name]', container);
+    expect(items.length).to.eq(6);
+    expect(
+      items.map((el, index) => el[index === 3 ? 'innerHTML' : 'textContent']),
+    ).to.deep.equal(['', 'Not selected', '', '', ',  ', '']);
+  });
+
+  it('should render the confirmation page with evidence', () => {
+    const data = getData();
+    const { container } = render(
+      <Provider store={mockStore(data)}>
+        <ConfirmationPageV2 />
+      </Provider>,
+    );
+
+    expect($('va-alert[status="success"]', container)).to.exist;
+    expect($('va-alert[status="success"]', container).innerHTML).to.contain(
+      `After we’ve completed our review`,
+    );
+    // expect($('va-loading-indicator', container)).to.exist;
+    const h2s = $$('h2', container);
+    expect(h2s.length).to.eq(5);
+    expect(h2s.map(el => el.textContent)).to.deep.equal([
+      // `You submitted your Board Appeal request on ${date}`,
```

Comment:
Curious what the commented-out h2s are about? Non-blocking, just want to make sure they should still be there.

Reply:
Oh, in our NOD form, the H2 inside the `va-alert` has a date inside. I was waiting for content recommendations, but I think we can get this work pushed out before additional changes are needed. I should have removed this comment.